util/execdetails/execdetails.go

```diff
@@ -410,6 +412,16 @@ func (e *RuntimeStats) SetRowNum(rowNum int64) {
 	atomic.StoreInt64(&e.rows, rowNum)
 }
 
+// SetConcurrencyInfo sets the concurrency information.
+func (e *RuntimeStats) SetConcurrencyInfo(concurrencyInfo string) {
+	var val atomic.Value
+	val.Store(concurrencyInfo)
+	e.concurrency = val.Load().(string)
```

Comment:
It cannot prevent data race...

Reply:
Update. Use the mutex to prevent data race